php-frontend/src/main/java/org/sonar/php/parser/PHPGrammar.java

```diff
@@ -494,6 +504,23 @@ public UseStatementTree USE_STATEMENT() {
         EOS()));
   }
 
+  public UseStatementTree GROUP_USE_STATEMENT() {
```

Comment:
@pynicolas As discussed we will keep this behaviour. A test case to make it obvious is added. 6242e5a7ea84c5b0307d51fca09a0e77cc86ed52